server/services/database/users.js

```diff
@@ -31,7 +31,19 @@ const getUserByEmail = email => {
   });
 };
 
-const createUser = ({ email, password, name, role }) => {
+const getUserIdByEmail = email => {
+  return new Promise(resolve => {
+    pool.query(
+      "SELECT id FROM users where email = $1",
+      [email],
+      (error, result) => {
+        resolve(result.rows[0]);
+      }
+    );
+  });
+};
```

Comment:
Can you rewrite this without the new Promise syntax, but using pool.query(...).then(...)?